mavros_msgs/msg/ComponentStatus.msg

```diff
@@ -0,0 +1,16 @@
+# Mavros message: COMPONENTSTATUS
+
+std_msgs/Header header
+
+uint8 component               # See enum MAV_COMPONENT
+
+uint8 cpu_usage               # percentage
+int8 temperature              # deg celsius
+uint32 ram_free               # MB
```

Comment:
For me it is a little intuitive that you will want to know the ram_free rather than the ram_usage. It is the same thing, but slightly strange.